recommender/api/schemas/database_dump/user.py

```diff
@@ -13,6 +13,12 @@
             description="The unique identifier of a the user",
             example=1234,
         ),
+        "aai_token": fields.String(
+            required=True,
+            title="AAI Token",
+            description="AAI token of the user",
+            example="abcdefghijklmnop",
```

Comment:
This example is misleading and may cause confusion.
An example of an uid can be: 64-characters@egi.eu
General structure: [id-hex]@[sso-provider]